src/internal/views/nunjucks/billing/batch-invoice.njk

```diff
@@ -72,18 +72,85 @@ invoiceDebitsTotal %}
 
       {{ batchMetadata(batch, invoice)}}
 
+      <!-- Bills linked to reissue -->
+      {% macro displayBillingState(inv, billingBatchId) %}
+        {% if inv.rebillingStateLabel == 'rebill' or inv.rebillingStateLabel == 'rebilled'  %}
+          Reissued <span class="screen-reader-only">bill</span>
+        {% endif %}
+
+        {% if inv.rebillingStateLabel == 'original' %}
+          Original <span class="screen-reader-only">bill</span>
+        {% endif %}
+
+        {% if inv.rebillingStateLabel == 'reversal' and inv.originalInvoiceId != null %}
```

Comment:
this 
`{% if inv.rebillingStateLabel == 'reversal' and inv.originalInvoiceId != null %}`
can change to
`{% if inv.rebillingStateLabel == 'reversal' %}`
because if the label is reversal it is the reversal and the originalInvoiceId can never be null